src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -514,12 +514,13 @@ object ComposterOverlay {
         }
 
         ChatUtils.sendCommandToServer("gfs ${internalName.asString()} ${itemsNeeded - havingInInventory}")
-        if (itemsNeeded > havingInInventory - havingInSacks) {
+        val havingInTotal = havingInInventory + havingInSacks
+        if (itemsNeeded >= havingInTotal) {
```

Comment:
Why is this more than or equal to now?
They should be notified that they've run out in sacks once the need it again imo

Reply:
So they know to get more box of seeds (or whatever).

Reply:
i still dont see the need to notify them before they need more 🤷

Reply:
I use buy orders, so that way I can make a buy order when I know i'll need more, but i don't have to wait for buy order to fill if I don't notice until I need to refill composter again.

Reply:
i guess